src/collatzConjecture.js

```diff
@@ -0,0 +1,15 @@
+export default function collatzConjecture(n){
+  let collatzArr = []
+  collatzArr.push(n)
```

Comment:
Also, I would use whitespace between initial setup (the function's equivalent of an introductory paragraph in an essay), and the body of the function.